package.json

```diff
@@ -1,55 +1,27 @@
 {
-  "name": "ally",
+  "name": "tokenator",
```

Comment:
you might want to name it something like `@labs-solo/tokenator`

Reply:
done